tests/test_cube/test_cube.py

```diff
@@ -54,6 +53,30 @@ def test_segy_no_file_exception():
         Cube().scan_segy_traces("not_a_file", outfile="not_relevant")
 
 
+def test_segy_export_import(tmpdir):
+    cube = xtgeo.cube_from_file(SFILE4)
+    assert cube.zinc == 4
+
+    # change to study rounding effect
+    cube.zinc = 3.99
+    fname = join(tmpdir, "small1.segy")
+    cube.to_file(fname, fformat="segy")
+    cube2 = xtgeo.cube_from_file(fname)
+
+    assert cube.values.mean() == pytest.approx(cube2.values.mean())
```

Comment:
Instead of testing mean, you could test all values by using `cube.values.tolist() == pytest.approx(cube2.values.tolist())

Reply:
Think this is even better: `np.testing.assert_equal(cube.values, cube2.values)`